ocaml/xapi/xapi_clustering.ml

```diff
@@ -19,10 +19,9 @@ open D
 
 (* Called by Cluster.create/destroy and Cluster_host.enable/disable *)
 let set_ha_cluster_stack_if_ha_enabled ~__context =
```

Comment:
Please rename this function, you dropped the `if` below, drop it from the name too.